prototype_source/openvino_quantizer.rst

```diff
@@ -0,0 +1,248 @@
+PyTorch 2 Export Quantization for OpenVINO torch.compile backend.
+===========================================================================
+
+**Authors**: `Daniil Lyakhov <https://github.com/daniil-lyakhov>`_,  `Aamir Nazir <https://github.com/anzr299>`_,  `Alexander Suslov <https://github.com/alexsu52>`_, `Yamini Nimmagadda <https://github.com/ynimmaga>`_, `Alexander Kozlov <https://github.com/AlexKoff88>`_
+
+Prerequisites
+--------------
+- `PyTorch 2 Export Post Training Quantization <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html>`_
+- `How to Write a Quantizer for PyTorch 2 Export Quantization <https://pytorch.org/tutorials/prototype/pt2e_quantizer.html>`_
+
+Introduction
+--------------
+
+**This is an experimental feature, the quantization API is subject to change.**
```

Comment:
Just a suggestion, but I'd put this in a note callout: `.. note::`

Reply:
Thanks! Done